config/application.rb

```diff
@@ -1,6 +1,7 @@
 require File.expand_path('../boot', __FILE__)
 
 require 'rails/all'
+require 'elasticsearch/rails/instrumentation'
```

Comment:
The docs say to use `application.rb` and for some reason it only works there. If we can figure out a better way I'm game but it doesn't work in the initializer right now.